src/riak_core_console.erl

```diff
@@ -752,6 +753,27 @@ commit_staged([]) ->
                       "plan and try to commit again~n")
     end.
 
+timezone([]) ->
+    TZ = riak_core_metadata:get({riak_core, timezone}, string,
+                                [{default, "not configured"}]),
+    io:format("~s~n", [TZ]);
+timezone([TZString]) ->
+    TZ = jam:process(jam_iso8601:parse_tz(TZString)),
+    attempt_set_timezone(jam:is_valid(TZ), TZ, TZString).
```

Comment:
I would rather have `jam_iso8601:parse_tz/1` either return the standard `{ok, TZ} | {error, Reason}`, or return `TZ` and throw a `badarg` if it's fed garbage. This way, you would avoid the need to have `TZ` validated separately and pass the validation result as a separate arg to `attempt_set_timezone` (and incidentally, dialyzer would produce a clearer message then).